faster_whisper/feature_extractor.py

```diff
@@ -142,11 +142,11 @@ def stft(
     hop_length: int = None,
     win_length: int = None,
     window: np.ndarray = None,
-    center=True,
-    mode="reflect",
-    normalized=False,
-    onesided=None,
-    return_complex=None,
+    center: bool = True,
+    mode: str = "reflect",
+    normalized: bool = False,
+    onesided: bool = None,
+    return_complex: bool = None,
```

Comment:
The name "input tensor" should be renamed in multiple places. `input_array` seems to be a better name as the inputs are no longer tensors.

Reply:
Will do